src/java/generator/MapGenerator.java

```diff
@@ -21,33 +21,58 @@
 import java.security.MessageDigest;
 import java.security.NoSuchAlgorithmException;
 import java.util.*;
-import java.util.concurrent.ExecutionException;
+import java.util.stream.Stream;
 
 public strictfp class MapGenerator {
 
 	public static final boolean DEBUG = false;
-
 	public static final String VERSION = "1.0.1";
-
 	private static final Random SEED_GEN = new Random();
 
 	//read from cli args
-	private static String FOLDER_PATH = ".";
-	private static String MAP_NAME = "debugMap";
-	private static long SEED = SEED_GEN.nextLong();
-	private static Random RANDOM = new Random(SEED);
-	private static Optional<Biome> BIOME = Optional.empty();
+	private static String folderPath = ".";
+	private static String mapName = "debugMap";
+	private static long seed = SEED_GEN.nextLong();
```

Comment:
I'd remove the SEED_GEN and just create a new Random() inline.

Reply:
Makes sense. Done